### PR TITLE
fill probeplugin map and comment correct

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:
filling probedplugin map  after init and correct comment.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```NONE

```

### CODE REVIEW

File: pkg/volume/plugins.go [hunk 1]
```diff
@@ -483,7 +483,7 @@ func (pm *VolumePluginMgr) initProbedPlugin(probedPlugin VolumePlugin) error {
 	if err != nil {
 		return fmt.Errorf("Failed to load volume plugin %s, error: %s", name, err.Error())
 	}
-
+	pm.probedPlugins[name] = probedPlugin
```
Comment (jsafrane) [inline pkg/volume/plugins.go]: `pm.probedPlugins[name]` is filled in `pm.refreshProbedPlugins`, just after `initProbedPlugin` returns. Do I miss anything?

Can you please post some sequence of operations where `pm.probedPlugins[name] = probedPlugin` on this place is necessary?

**Comment by davidz627** [comment]
/close
due to inactivity

**Pull request closed without merging**